daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/dfa/TextParser.scala

```diff
@@ -54,7 +54,7 @@ class TextParser(override val context: TermRuntimeData)
         if (isDelimRequired) Nope
         else {
           val totalNumCharsRead = 0
-          input.getString(totalNumCharsRead, state)
+          input.getSomeString(totalNumCharsRead, state)
```

Comment:
This behavior is slightly different, this now doesn't care if it doesn't get totalNumCharsRead--before it would have been an error. Maybe that wasn't possible in this code path? It's hard to tell.

Also a bit concerning that this change has no test coverage. Maybe this code isn't even possible to reach?

Reply:
I do think it is not reachable any more. I will look at it again, and if it's really dead code I'll remove.